doc/dev_guide/internals/iproto/images/repl_ballot.puml

```diff
@@ -8,13 +8,15 @@ skinparam {
 }
 
 json "**IPROTO_BALLOT body**" as ballot {
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_IS_RO_CFG]]": "MP_BOOL",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_VCLOCK]]": "MP_MAP",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_GC_VCLOCK]]": "MP_MAP",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_IS_RO]]": "MP_BOOL",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_IS_ANON]]": "MP_BOOL",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_IS_BOOTED]]": "MP_BOOL",
-    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys IPROTO_BALLOT_CAN_LEAD]]": "MP_BOOL"
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_IS_RO_CFG]]": "MP_BOOL",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_VCLOCK]]": "MP_MAP",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_GC_VCLOCK]]": "MP_MAP",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_IS_RO]]": "MP_BOOL",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_IS_ANON]]": "MP_BOOL",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_IS_BOOTED]]": "MP_BOOL",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_CAN_LEAD]]": "MP_BOOL",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_BOOTSTRAP_LEADER_UUID]]": "MP_STR",
+    "[[https://tarantool.io/en/doc/latest/dev_guide/internals/iproto/keys/#general-replication IPROTO_BALLOT_REGISTERED_REPLICA_UUIDS]]": "MP_ARRAY"
```

Comment:
Should we describe the types of array elements somehow?

Like "MP_ARRAY of MP_STR elements"?
Although we don't do that for MP_MAP (vclock).

Reply:
Updated description